src/browser/modules/Frame/styled.tsx

```diff
@@ -18,47 +18,26 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Mostly added proper typings to these components so the ignore-comments could be removed.